src/TensorAlgebra.jl

```diff
@@ -2,6 +2,8 @@ module TensorAlgebra
 
 export contract, contract!, eigen, eigvals, lq, left_null, qr, right_null, svd, svdvals
```

Comment:
I didn't get the chance to comment on this before, but I think the export list also needs to be updated